File: Iniciante/URI 1010.lua
line1 = {}
n = 0
for i in string.gmatch(io.read("*l"), "%S+") do
   n = n + 1
   line1[n] = i
end
cod1 = tonumber(line1[1])
n1 = tonumber(line1[2])
valor1 = tonumber(line1[3])

line2 = {}
n = 0
for i in string.gmatch(io.read("*l"), "%S+") do
   n = n + 1
   line2[n] = i
end
cod2 = tonumber(line2[1])
n2 = tonumber(line2[2])
valor2 = tonumber(line2[3])

total = (n1 * valor1) + (n2 * valor2)
print(string.format('VALOR A PAGAR: R$ %.2f', total))

File: Iniciante/URI 1010.clj
(require '[clojure.string :as str])

(def line1 (str/split (read-line) #" "))
(def cod1 (Integer/parseInt (get line1 0)))
(def n1 (Integer/parseInt (get line1 1)))
(def valor1 (Float/parseFloat (get line1 2)))

(def line2 (str/split (read-line) #" "))
(def cod2 (Integer/parseInt (get line2 0)))
(def n2 (Integer/parseInt (get line2 1)))
(def valor2 (Float/parseFloat (get line2 2)))

(def total (+ (* n1 valor1) (* n2 valor2)))

(printf "VALOR A PAGAR: R$ %.2f\n" total)